test/functional/tests/fault_injection/test_fault_injection_many_to_one.py

```diff
@@ -0,0 +1,252 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm, casadm_parser
+from api.cas.cache_config import CacheMode, CleaningPolicy, SeqCutOffPolicy
+from cas import cli
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_utils.size import Size, Unit
+
+block_size = Size(1, Unit.Blocks4096)
+blocks_100MiB = int(Size(100, Unit.MebiByte).value / block_size.value)
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode)
```

Comment:
I think that there is no reason to run this test with parametrize. It would multiple tests to run and give no additional input. In my opinion it can be removed here